src/backend/metal/Cargo.toml

```diff
@@ -34,12 +34,9 @@ metal = { git = "https://github.com/gfx-rs/metal-rs", rev="ba08f5f98c70ab941020b
 foreign-types = "0.3"
 objc = "0.2.5"
 block = "0.1"
-#TODO: remove this, only used for `Id`
-cocoa-foundation = "0.1"
```

Comment:
Now that I'm looking at this again, we should probably keep using it. My comment got you fixing this, sorry!
The justification is that `metal-rs` relies on cocoa-foundation, and as long as it's there, we gain nothing from trying to re-implement the types from it in `gfx-backend-metal`.

Reply:
Done :smile: